packages/connect/src/utils/isPhishingDomain.ts

```diff
@@ -0,0 +1,4 @@
+// @ts-expect-error (js-only dependency)
+import checkForPhishing from 'eth-phishing-detect';
+
+export const isPhishingDomain = (domain: string) => checkForPhishing(domain);
```

Comment:
Perhaps just add typings for `eth-phishing-detect`?

Reply:
what would be your prefered way of doing this? I felt like it is not worth doing since it is just super simple lib and I make sure it returns what it should return (boolean) in unit tests

Reply:
in `types/phishing.d.ts`

```
declare module 'eth-phishing-detect' {
    export default function isPhishingDomain(domain: string): boolean;
}
```

Reply:
cool. done